app/src/main/java/de/xikolo/controllers/helper/ShortcutHelper.kt

```diff
@@ -0,0 +1,62 @@
+package de.xikolo.controllers.helper
```

Comment:
It is strictly speaking not a helper for a controller. I think the `utils` package would be more fitting. Mind to rename then.